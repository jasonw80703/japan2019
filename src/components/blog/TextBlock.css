.text-block {
  margin: 1% 10% 1% 10%;
}
