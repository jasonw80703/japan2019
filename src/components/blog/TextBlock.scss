.text-block {
  margin: 1% 10% 1% 10%;
  font-size: 120%;
}

.text-block-with-image {
  font-size: 120%;
}

.image-left {
  margin-right: 18%;
}

@media (max-width: 975px) {
  .text-block-with-image {
    margin: 1% 10% 1% 10% !important;
  }
}

.image-right {
  margin-left: 18%;
}