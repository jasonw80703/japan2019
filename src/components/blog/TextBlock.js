import React, { Component } from 'react';
import PropTypes from 'prop-types';
import './TextBlock.css';

export default class TextBlock extends Component {
  render() {
    const { text } = this.props;
    return (
      <p className='text-block'>{text}</p>
    );
  }
};

TextBlock.propTypes = {
  text: PropTypes.string.isRequired,
};