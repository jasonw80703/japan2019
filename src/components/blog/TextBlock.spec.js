import React from 'react';
import renderer from 'react-test-renderer';
import TextBlock from './TextBlock';

describe('<TextBlock />', () => {
  const defaultProps = {
    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.'
  };

  it('renders as expected', () => {
    const tree = renderer
      .create(<TextBlock {...defaultProps} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});