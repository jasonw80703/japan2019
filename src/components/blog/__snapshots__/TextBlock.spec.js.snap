// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextBlock /> renders as expected 1`] = `
<p
  className="text-block"
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
</p>
`;
